### DISCUSSION TITLE
Using `rkyv` with `axum`

### DISCUSSION THREAD

**Discussion by thestarmaker** [tokio-rs/axum#2336] [Q&A]
Hi guys, I've been looking for an example of how to write `rkyv` encoded response with Axum while avoiding additional copying. After serialisation `rkyv` leaves users with `AlignedVec` structire (an aligned `[u8]` of a sort), is there a way to just provide this as a response to Axum without copying?

**Comment by davidpdrsn** [comment]
I've never used `rkyv` so I'm not sure I can help there. Consider asking in the [tokio discord](https://discord.gg/tokio) instead. There are more people there so maybe someone has experience with it.
